spec/presenters/partially_signed_in_modal_presenter_spec.rb

```diff
@@ -1,19 +1,30 @@
 require 'rails_helper'
 
 describe PartiallySignedInModalPresenter do
-  include ActionView::Helpers::TagHelper
+  include ActionView::Helpers::SanitizeHelper
 
-  let(:time_left_in_session) { 10 }
-  subject(:presenter) { PartiallySignedInModalPresenter.new(time_left_in_session) }
+  let(:expiration) { Time.zone.now + 1.minute + 1.second }
+  subject(:presenter) { PartiallySignedInModalPresenter.new(expiration) }
+
+  around do |ex|
+    freeze_time { ex.run }
+  end
 
   describe '#message' do
     it 'returns the partially signed in message' do
-      message = t(
+      expect(strip_tags(presenter.message(ActionController::Base.new))).to eq t(
         'notices.timeout_warning.partially_signed_in.message_html',
-        time_left_in_session: content_tag(:span, time_left_in_session, id: 'countdown'),
+        time_left_in_session: "1 minute and 1 second\n",
```

Comment:
is the trailing newline meaningful?

Reply:
Only in the sense that it's left over from the `strip_tags`. For reasons I'm not aware of off the top of my head, the output of rendering the component results in an extra newline being added.

Example:

```rb
"You will be signed out in <lg-countdown data-expiration=\"2022-03-02T07:12:34-12:00\" data-update-interval=\"30000\" data-start-immediately=\"false\">1 minute and 1 second</lg-countdown>\n. Select “keep me signed in” to stay logged in."
```

Reply:
Ah, I think the newline is probably the end-of-file newline technically on line 11 of [`countdown_component.html.erb`](https://github.com/18F/identity-idp/blob/efb324e591977b9b37c755cd1e752f6144baf0d2/app/components/countdown_component.html.erb).

Reply:
We could also just call `.chomp` on the string in the tests, or use our friends `<%- -%>` but really it's not worth much effort, like you said

Reply:
`<%=- -%>` wasn't enough, so updated to `#call` in [83102cb](https://github.com/18F/identity-idp/pull/6023/commits/83102cbef21eee8201fac80a4b6ca20f4be68e3f).